examples/games/contributors.rs

```diff
@@ -70,23 +62,21 @@ struct Velocity {
 const GRAVITY: f32 = 9.821 * 100.0;
 const SPRITE_SIZE: f32 = 75.0;
 
-const SATURATION_DESELECTED: f32 = 0.3;
-const LIGHTNESS_DESELECTED: f32 = 0.2;
-const SATURATION_SELECTED: f32 = 0.9;
-const LIGHTNESS_SELECTED: f32 = 0.7;
-const ALPHA: f32 = 0.92;
+const SELECTED: Hsla = Hsla::hsl(0.0, 0.9, 0.7);
```

Comment:
Much nicer! Only suggestion is Oklch might look nicer since it has better uniformity

Reply:
I'm honestly not sure how to use Oklch for this. Maybe I read up a bit more. I naively converted these values to Oklch and kept the rest of the code the same and some of them look okay but most look dark and saturated.